projects/storefrontlib/cms-components/navigation/search-box/search-box.component.ts

```diff
@@ -163,8 +201,20 @@ export class SearchBoxComponent implements OnInit, OnDestroy {
    * Opens the type-ahead searchBox
    */
   open(): void {
-    this.searchBoxComponentService.toggleBodyClass(SEARCHBOX_IS_ACTIVE, true);
-    this.searchBoxActive = true;
+    // TODO: (CXSPA-6929) - Remove feature flag next major release
```

Comment:
I had a look at the cleanup ticket and it mentions removing the feature flags. Could we please also include making service injections no-longer optional?